src/pages/authpages/Login.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./login.css";
+
+function Login() {
+  return (
+    <div class="login-container">
+      <img
+        class="login-img"
```

Comment:
please convert all `class` attributes to `className`

Reply:
Done. Thanks for reminding